<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config_window</class>
 <widget class="QMainWindow" name="Config_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config.json settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="engine_ver_edit"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="ok_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search file list</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="max_response_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Max responses</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="engine_name_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="engine_ver_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Engine version</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="engine_name_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Engine name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="cancel_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="max_response_spin">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="add_file_button">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="remove_file_button">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QListView" name="file_listView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>Config_window</receiver>
   <slot>clickedOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Config_window</receiver>
   <slot>clickedCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_file_button</sender>
   <signal>clicked()</signal>
   <receiver>Config_window</receiver>
   <slot>clickedRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_file_button</sender>
   <signal>clicked()</signal>
   <receiver>Config_window</receiver>
   <slot>clickedAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedOk()</slot>
  <slot>clickedCancel()</slot>
  <slot>clickedAdd()</slot>
  <slot>clickedRemove()</slot>
  <slot>maxResponsesChanged()</slot>
  <slot>engineNameChanged()</slot>
  <slot>engineVerChanged()</slot>
 </slots>
</ui>
